lib/core/base/check.js

```diff
@@ -58,6 +58,7 @@ Check.prototype.enabled = true;
  * @param  {Function} callback Function to fire when check is complete
  */
 Check.prototype.run = function(node, options, context, resolve, reject) {
+	/* eslint max-statements:0 */
```

Comment:
This seems unnecessary.

Reply:
The "max statements" rule seems unnecessary. We always increase the minimum rather than refactoring our code. IMO it should be disabled.